lib/rhc/helpers.rb

```diff
@@ -132,6 +132,7 @@ def user_agent
       debug certificate_key(value)
     end
 
+    global_option '--ssh STRING', 'Path to a ssh executable and command options to use when performing ssh commands. When providing options, ensure path and options are encased in single or double quotes.', :hide => true
```

Comment:
Verified, this option can be passed to `rhc setup` and will set the `ssh` option in `~/.openshift/express.conf` appropriately.